edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionManager.cs

```diff
@@ -29,13 +29,15 @@ public class ConnectionManager : IConnectionManager
         readonly ICredentialsCache credentialsCache;
         readonly IIdentityProvider identityProvider;
         readonly IDeviceConnectivityManager connectivityManager;
+        readonly bool closeCloudConnectionWhenCloseDeviceConnection;
```

Comment:
Test case?

Reply:
Add one test for RemoveDeviceConnection method.